environments/common.yaml.gotmpl

```diff
@@ -79,10 +105,84 @@ postgresqlHA:
     database: api
     repmgrPassword: such-secret
 
+qkm:
+  enabled: true
+  url: "http://quorum-key-manager-quorumkeymanager:8080"
+  proto: https
+  fullname: "quorumkeymanager"
+  orchestrate:
+    storeName: "eth-accounts"
+    apiKey: {{ env "QKM_API_KEY" | default "NmwyMCB3YXMgaGVyZQ==" }}
```

Comment:
Does it make sense to have sensitive information such as API_KEY in default values?

Reply:
As it is stated in the README file every sensitive information should be identified and updated/removed by the end user. Having these values is just for informative purpose, so I see no problem with that. At least it provides info on the expected format.